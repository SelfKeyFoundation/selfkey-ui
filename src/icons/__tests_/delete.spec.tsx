import * as React from 'react';
import { create } from 'react-test-renderer';
import DeleteIcon from '../delete';

describe('Delete Icon', () => {
  it('should show Delete Icon', () => {
    const component = create(
      <DeleteIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});