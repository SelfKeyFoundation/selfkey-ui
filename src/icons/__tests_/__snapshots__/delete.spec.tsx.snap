// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Icon should show Delete Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root Component-root-0-2-1"
  focusable="false"
  version="1.1"
  viewBox="0 0 16 16"
>
  <defs />
  <g
    fillRule="evenodd"
    id="-Main-Dashboard"
    stroke="none"
    strokeWidth="1"
  >
    <g
      id="304-Manage-My-Tokens"
      transform="translate(-1244.000000, -597.000000)"
    >
      <g
        id="Table"
        transform="translate(150.000000, 442.000000)"
      >
        <g
          id="Col7"
          transform="translate(1094.000000, 155.000000)"
        >
          <path
            d="M9.13137085,8 L10.8284271,9.69705627 L9.69705627,10.8284271 L8,9.13137085 L6.30294373,10.8284271 L5.17157288,9.69705627 L6.86862915,8 L5.17157288,6.30294373 L6.30294373,5.17157288 L8,6.86862915 L9.69705627,5.17157288 L10.8284271,6.30294373 L9.13137085,8 Z M8,14.4 C4.4712,14.4 1.6,11.5288 1.6,8 C1.6,4.4712 4.4712,1.6 8,1.6 C11.5288,1.6 14.4,4.4712 14.4,8 C14.4,11.5288 11.5288,14.4 8,14.4 L8,14.4 Z M8,0 C3.5816,0 0,3.5816 0,8 C0,12.4184 3.5816,16 8,16 C12.4184,16 16,12.4184 16,8 C16,3.5816 12.4184,0 8,0 L8,0 Z"
            id="icon-delete"
          />
        </g>
      </g>
    </g>
  </g>
</svg>
`;
